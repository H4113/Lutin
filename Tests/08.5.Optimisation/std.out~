const B = 8;
var a;
a := 192;
ecrire 193;
a := 45;
ecrire 45;
