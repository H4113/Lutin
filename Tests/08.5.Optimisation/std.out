var a;
var b;
var c;
var d;
var e;
lire a;
lire b;
lire c;
lire d;
e := ((a+b)*(c+d));
